From 47f4b4a24c85cf5ba0f3091a626af523c9207f84 Mon Sep 17 00:00:00 2001
From: Nicolai Buchwitz <n.buchwitz@kunbus.com>
Date: Wed, 2 Feb 2022 16:18:52 +0100
Subject: [PATCH 1371/2201] keep disabled node hdmi1 in devicetree

The second hdmi port on cm4s is disabled by default (default from
bcm2711-rpi.dtsi) and therefore it is not necessary to remove it in the
specific cm4s overlay (as done in 64328ab674dc3e3f0356d066916546f6b2876257)
---
 arch/arm/boot/dts/bcm2711-rpi-cm4s.dts | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm/boot/dts/bcm2711-rpi-cm4s.dts b/arch/arm/boot/dts/bcm2711-rpi-cm4s.dts
index c0808eca28b3..cefef46bd866 100644
--- a/arch/arm/boot/dts/bcm2711-rpi-cm4s.dts
+++ b/arch/arm/boot/dts/bcm2711-rpi-cm4s.dts
@@ -206,8 +206,6 @@
 #include "bcm283x-rpi-csi1-4lane.dtsi"
 #include "bcm283x-rpi-i2c0mux_0_28.dtsi"
 
-/delete-node/ &hdmi1;
-
 / {
 	chosen {
 		bootargs = "coherent_pool=1M 8250.nr_uarts=1 snd_bcm2835.enable_compat_alsa=0 snd_bcm2835.enable_hdmi=1";
-- 
2.17.1

